modules/dashboard/src/components/StatsSummary.js

```diff
@@ -8,27 +8,40 @@ const TopGrid = styled(Grid)({
   flexDirection: "row",
   width: "100%",
   height: "100%",
-  justifyContent: "center",
+  justifyContent: "flex-start",
   alignItems: "center",
-})
+});
 
-const StatTypography = styled(Typography)({
-  textAlign: "center",
-  width: "90%",
+const StatLabelTypography = styled(Typography)({
+  alignSelf: "center",
+  textAlign: "left",
   fontSize: "24px",
+  fontWeight: "600",
   color: "#002868",
   textDecoration: "none",
-})
+});
+
+const StatTypography = styled(Typography)({
+  fontSize: "24px",
+  fontWeight: "400",
+});
 
 const ErrorTypography = styled(Typography)({
   color: "red",
-})
+});
 
+const RefreshButton = styled(Button)({
+  marginLeft: "5%",
+});
+
+const SectionGrid = styled(Grid)({
+  marginLeft:"5%"
+})
 
 const address = {
   mainnet: "0xf3f722f6ca6026fb7cc9b63523bbc6a73d3aad39", //"0xF80fd6F5eF91230805508bB28d75248024E50F6F", //,
-  // staging: "0x0f41a9aaee33d3520f853cb706c24ca75cac874e",
-  // rinkeby: "0x5307b4f67ca8746562a4a9fdeb0714033008ef4a",
+  staging: "0x5307B4F67ca8746562A4a9fdEb0714033008Ef4A",
+  // rinkeby: "0xDA3CCBa9F3e3a9fE7D0Ed9F699Ca2BEF78Ba7A6c",
 };
```

Comment:
looks like were using this in a couple of places, we should add this at a global scope or in a `constants` file

Reply:
should this block merging? can definitely change moving forward, but this isn't a substantive change from what's on staging right now

Reply:
bump^